screwdriver/screwdriver.go

```diff
@@ -435,3 +445,24 @@ func (a api) UpdateStepStop(buildID, stepName string, exitCode int) error {
 
 	return nil
 }
+
+func (a api) SecretsForBuild(build Build) (Secrets, error) {
+	u, err := a.makeURL(fmt.Sprintf("builds/%s/secrets", build.ID))
```

Comment:
Just to clarify, does this mean we need an API endpoint like `GET build/{id}/secrets`?